packages/web/src/lib/web-mode.ts

```diff
@@ -282,4 +286,75 @@ export class WebMode {
 		}
 		throw new Error("Transaction Error");
 	}
+
+	/**
+	 * Creates an algosdk.Transaction object based on execParams and suggestedParams
+	 * @param execParams execParams containing all txn info
+	 * @param txParams suggestedParams object
+	 * @returns array of algosdk.Transaction objects
+	 */
+	makeTx(execParams: ExecParams[], txParams: algosdk.SuggestedParams): Transaction[] {
+		const txns: Transaction[] = [];
+		for (const [_, txn] of execParams.entries()) {
+			txns.push(mkTransaction(txn, txParams));
+		}
+		return txns;
+	}
+
+	/**
+	 * Signs a Transaction object
+	 * @param transaction transaction object.
+	 * @returns SignedTransaction
+	 */
+	async signTx(transaction: algosdk.Transaction): Promise<SignedTransaction> {
+		const binaryTx = transaction.toByte();
+		const base64Tx = this.algoSigner.encoding.msgpackToBase64(binaryTx);
+		const signedTx = await this.algoSigner.signTxn([
```

Comment:
you can use `signTransaction` as it is already implemented in web-mode

Reply:
These functions have different interface. The goal is to provide user with methods that will work the same across all deployers

Reply:
Yes but you can reuse the function `signTransaction` like I suggest here
```suggestion
		const signedTx = await this.signTransaction([
```